src/components/Pagination/Pagination.js

```diff
@@ -28,7 +28,7 @@ export const Pagination = ({ postsPerPage, totalPosts, paginate, currentPage, el
               </Button>
             </div>
           ))}
-        </div>
+        </section>
       </nav>
     );
   }
```

Comment:
Semicolons (in imports) - one convention, is there tabIndex prop in Button component?